clarity_ext/dilute_epp/perform_dilution.py

```diff
@@ -0,0 +1,164 @@
+# Calculates dilute volumes, volume from sample and volume from buffer
```

Comment:
This script could be refactored as an extension, that can be executed with the command `clarity-ext extension ...`, where the boilerplate could would go down to some base class that uses a context object. I will add some notes on that in the following comments.